project/EnsimeBuild.scala

```diff
@@ -154,7 +155,10 @@ object EnsimeBuild {
       ensimeUnmanagedSourceArchives += (baseDirectory in ThisBuild).value / "openjdk-langtools/openjdk8-langtools-src.zip",
       libraryDependencies ++= Seq(
         "com.h2database" % "h2" % "1.4.193",
-        "com.typesafe.slick" %% "slick" % "3.1.1",
+        "com.typesafe.slick" %% "slick" % (scalaBinaryVersion.value match {
```

Comment:
Should use CrossVersion.partial version over scalaBinaryVersion for the community build, btw.